app/models/bookmark.rb

```diff
@@ -0,0 +1,4 @@
+class Bookmark < ApplicationRecord
+  belongs_to :question
```

Comment:
has_one? or belongs to question

Reply:
replacing `belongs_to` with `has_one` raises `ActiveRecord::HasManyThroughCantAssociateThroughHasOneOrManyReflection: Cannot modify association 'User#bookmark_questions' because the source reflection class 'Question' is associated to 'Bookmark' via :has_one.
`

https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association

User <-> Question have a many to many relationship via `Bookmark`. Maybe we need a better name